src/main/g8/$name__norm$-impl/src/test/java/$package$/impl/$name__Camel$AggregateTest.java

```diff
@@ -0,0 +1,66 @@
+package $package$.impl;
+
+import akka.actor.testkit.typed.javadsl.TestKitJunitResource;
+import akka.actor.testkit.typed.javadsl.TestProbe;
+import akka.actor.typed.ActorRef;
+import akka.cluster.sharding.typed.javadsl.EntityContext;
+import $package$.impl.$name;format="Camel"$Command.Hello;
+import $package$.impl.$name;format="Camel"$Command.UseGreetingMessage;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import java.util.UUID;
+
+public class $name;format="Camel"$AggregateTest {
+    private static final String inmemConfig =
+        "akka.persistence.journal.plugin = \"akka.persistence.journal.inmem\" \n";
+
+    private static final String snapshotConfig =
+        "akka.persistence.snapshot-store.plugin = \"akka.persistence.snapshot-store.local\" \n"
+            + "akka.persistence.snapshot-store.local.dir = \"target/snapshot-"
+            + UUID.randomUUID().toString()
+            + "\" \n";
+
+    private static final String config = inmemConfig + snapshotConfig;
+
+    @ClassRule
+    public static final TestKitJunitResource testKit = new TestKitJunitResource(config);
+
+    @Test
+    public void testHello() {
+
+        String id = "Alice";
+        ActorRef<$name;format="Camel"$Command> ref =
+            testKit.spawn(
+                $name;format="Camel"$Aggregate.create(
+                    new EntityContext($name;format="Camel"$Aggregate.ENTITY_TYPE_KEY, id,  null)
+                )
+            );
+
```

Comment:
Unfortunately the only way I found to instantiate the Aggregate was passing a fake `EntityContext` with a `null` on the `replyTo` field that represents the `shardRegion: ActorRef`. :-(

Reply:
Yep, that's unfortunate. It would be nice we could have the means to create an `ActorRef[T]` that is intended to not be used, like a `NotUsed`, but then with the right `T` type.

Reply:
Can we add a comment explaining why we are passing `null` and confirming that this field won't be used?